usr/lib/common/key.c

```diff
@@ -2656,6 +2656,117 @@ CK_RV rsa_priv_unwrap_get_data(TEMPLATE *tmpl,
     return rc;
 }
 
+static CK_RV ibm_pqc_keyform_mode_attrs_by_mech(CK_MECHANISM_TYPE mech,
+                                                CK_ATTRIBUTE_TYPE *keyform_attr,
+                                                CK_ATTRIBUTE_TYPE *mode_attr,
+                                                const struct pqc_oid **oids)
+{
+    switch (mech) {
+    case CKM_IBM_DILITHIUM:
+        *keyform_attr = CKA_IBM_DILITHIUM_KEYFORM;
+        *mode_attr = CKA_IBM_DILITHIUM_MODE;
+        *oids = dilithium_oids;
+        break;
+    case CKM_IBM_KYBER:
+        *keyform_attr = CKA_IBM_KYBER_KEYFORM;
+        *mode_attr = CKA_IBM_KYBER_MODE;
+        *oids = kyber_oids;
+        break;
+    default:
+        TRACE_ERROR("Unsupported mechanims\n");
```

Comment:
Print the mech type in the TRACE_ERROR?

Reply:
Should only be called for Dilithium or Kyber, but sure, printing the mech makes sense.